internal/metamorph/metamorph_api/metamorph_api.proto

```diff
@@ -31,6 +31,7 @@ service MetaMorphAPI {
   rpc PutTransaction (TransactionRequest) returns (TransactionStatus) {}
   rpc PutTransactions (TransactionRequests) returns (TransactionStatuses) {}
   rpc GetTransaction (TransactionStatusRequest) returns (Transaction) {}
+  rpc GetManyTransactions (ManyTransactionsStatusRequest) returns (Transactions) {}
```

Comment:
How about the name `GetTransactions`? IMO that would sound better

Reply:
changed